src/main/java/org/folio/services/group/GroupFiscalYearTotalsService.java

```diff
@@ -125,14 +131,15 @@ private void updateGroupSummaryWithCalculatedFields(List<GroupFiscalYearTransact
       summary.withAvailable(available.doubleValue());
 
       BigDecimal expended = BigDecimal.valueOf(summary.getExpenditures());
-      summary.withCashBalance(totalFunding.subtract(expended).doubleValue());
+      BigDecimal credited = BigDecimal.valueOf(summary.getCredits());
+      summary.withCashBalance(totalFunding.subtract(expended).add(credited).doubleValue());
```

Comment:
regarding avaliable and unavaliable, I couldn't find calculation so, I didn't make changes here
https://github.com/folio-org/mod-finance/pull/252/files#diff-e3d35a5d97350527e0b5f76af072099d4f7fefa86ea729ba65fd661300bef79eR129-R130 (available and unavailable

Reply:
Unavailable is calculated based on the sum of the budgets' unavailable, which is correct. Available is calculated based on totalFunding and unavailable just above: this is not good in the rare case where one of the budgets' unavailable was changed from a negative value to 0. So we should recalculate available without using unavailable.

Reply:
Chagenged calculation of avaliable

Reply:
didn't touch the calculation of unavaliable, but seems it is possible to recalculate